README.md

```diff
@@ -136,12 +136,14 @@ connections and then processing each incoming HTTP request.
 
 The request object will be processed once the request headers have
 been received by the client.
-This request object implements the
+This request object uses an implementation of the
 [PSR-7 RequestInterface](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-7-http-message.md#32-psrhttpmessagerequestinterface)
+and implements the
+[PSR-7 ServerRequestInterface](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-7-http-message.md#321-psrhttpmessageserverrequestinterface)
```

Comment:
…implements the ServerRequestInterface which in turn extends the RequestInterface?